.github/ISSUE_TEMPLATE/BUG_REPORT_CUSTOM.md

```diff
@@ -42,6 +42,9 @@ Which version of JHipster are you using, is it a regression?
 <!-- What OS are you on? is this a problem with all browsers or only IE8? -->
 
 - [ ] Tickets opened without reproduction steps or that doesn't follows the template recommendation will be **closed**.
+- [ ] This issue is prompt related or an error that prevent jhipster from generating an application.
+- [ ] I don't have and JDL otherwise I should open an [JDL Issue](https://github.com/jhipster/generator-jhipster/issues/new?template=BUG_REPORT_JDL.md)
```

Comment:
```suggestion
- [ ] I don't have a JDL otherwise I should open an [JDL Issue](https://github.com/jhipster/generator-jhipster/issues/new?template=BUG_REPORT_JDL.md)
```